build.zig

```diff
@@ -1,8 +1,8 @@
 const std = @import("std");
 const builtin = @import("builtin");
 pub const gpu = @import("gpu/build.zig");
-const gpu_dawn = @import("gpu-dawn/build.zig");
-pub const glfw = @import("glfw/build.zig");
+const gpu_dawn = @import("gpu/libs/mach-gpu-dawn/build.zig");
+pub const glfw = @import("gpu/libs/mach-glfw/build.zig");
```

Comment:
did these symlink changes make it into here by accident?

Reply:
I was testing if it worked and forgot to go back to how it was before

Reply:
hopefully this fixes #405